src/options/getRepoOptions.ts

```diff
@@ -0,0 +1,40 @@
+import { cosmiconfigSync } from 'cosmiconfig';
+import { getDefaultRemoteBranch } from 'workspace-tools';
+import { RepoOptions, CliOptions } from '../types/BeachballOptions';
+
+export function getRepoOptions(cliOptions: CliOptions): RepoOptions {
```

Comment:
**suggestion** this is not a very good API, as we are lying to consumers, instead we should return proper interface or null

Reply:
As far as I can tell, the return type of this function actually ought to be `Partial<RepoOptions>` since specifying all the options in the config is never required. (Also all of the beachball option types are not the most crisp/accurate with their notion of required vs. optional properties.) But fixing that would potentially farther-reaching consequences than I want to deal with right now, since this isn't new code.